src/masternodeman.cpp

```diff
@@ -1000,9 +1000,10 @@ void CMasternodeMan::DoFullVerificationStep(CConnman& connman)
     rank_pair_vec_t vecMasternodeRanks;
     GetMasternodeRanks(vecMasternodeRanks, nCachedBlockHeight - 1, MIN_POSE_PROTO_VERSION);
 
-    LOCK(cs);
+    std::vector<CAddress> vAddr;
 
-    int nCount = 0;
+    {
```

Comment:
This brace does not appear to get closed... What am I missing?

Reply:
Yep, I tried to keep changes minimal here.

Reply:
I would think we should indent properly then. Personal preference I guess